src/main/java/dao/DBConfiguration.java

```diff
@@ -0,0 +1,60 @@
+package dao;
+
+import org.sqlite.SQLiteDataSource;
+
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.sql.Statement;
+
+public class DBConfiguration {
```

Comment:
Can you move this class to a `configuration` package?

Reply:
Done